demo/kubernetes/rook-storageclass.yaml

```diff
@@ -1,3 +1,18 @@
+apiVersion: rook.io/v1alpha1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
```

Comment:
the challenge of using the metadata name is if you want to create a pool of the same name in two different rook clusters.